docs/devel/on-call-build-cop.md

```diff
@@ -139,18 +139,17 @@ or [flaky test builds](https://goto.google.com/k8s-test/view/Flaky/) are not
 your responsibility to monitor. The `Test owner:` in the job description will be
 automatically emailed if the job is failing.
 
-* If you are a weekday oncall, ensure that PRs confirming to the following
+* If you are oncall, ensure that PRs confirming to the following
 pre-requisites are being merged at a reasonable rate:
 
   * [Have been LGTMd](https://github.com/kubernetes/kubernetes/labels/lgtm)
   * Pass Travis and Jenkins per-PR tests.
   * Author has signed CLA if applicable.
 
 
-* If you are a weekend oncall, [never merge PRs manually](collab.md), instead
-add the label "lgtm" to the PRs once they have been LGTMd and passed Travis;
-this will cause merge-bot to merge them automatically (or make them easy to find
-by the next oncall, who will merge them).
+* Although the shift schedule shows you as being scheduled Monday to Monday,
```

Comment:
is anyone from community on build cop (im guessing no because they can't access jenkins, but do we plan to extend the rotation)? if not we should consider moving this doc out of the upstream repo